src/server/game/Globals/ObjectMgr.cpp

```diff
@@ -4991,6 +4994,16 @@ void ObjectMgr::LoadQuests()
                 nextQuestItr->second.DependentPreviousQuests.push_back(qinfo->GetQuestId());
         }
 
+        if (uint32 breadcrumbForQuestId = std::abs(qinfo->_breadcrumbForQuestId))
+        {
+            if (_questTemplates.find(breadcrumbForQuestId) == _questTemplates.end())
+                TC_LOG_ERROR("sql.sql", "Quest %u is a breadcrumb for quest %u, but no such quest exists", qinfo->_id, breadcrumbForQuestId);
```

Comment:
should we set qinfo->_breadcrumbForQuestId to 0 in this case ? that way we don't have to check it anymore at runtime

Reply:
suppose you could, would remove https://github.com/TrinityCore/TrinityCore/blob/ea71f6bebc5c6b3097832b14a0540d7288c314a1/src/server/game/Entities/Player/Player.cpp#L15463